playbooks/roles/ocp-config/tasks/restart-config.yaml

```diff
@@ -0,0 +1,32 @@
+---
+# Add restart always configurations to the services
+- name: Create dropin directory
+  file:
+    path: /etc/systemd/system/{{item}}.service.d
+    state: directory
+    mode: 0755
+  become: true
+  loop:
+    - squid
```

Comment:
You need to add httpd and dhcpd. Also this could go into a variable instead of repeating in every task.

Reply:
Added restart always for httpd and dhcpd services and moved the the list of services under default/main/main.yaml.